pkl-core/src/main/java/org/pkl/core/RendererException.java

```diff
@@ -15,7 +15,7 @@
  */
 package org.pkl.core;
 
-public class RendererException extends RuntimeException {
+public final class RendererException extends RuntimeException {
```

Comment:
Is IntelliJ complaining about this, or is it an unrelated change?

Reply:
Unlike the other commits, this one doesn’t address IntelliJ warnings (see commit messages). But it also falls into the “cleanup” category: Generally, classes are final if possible, but some aren’t.